sensor_msgs/PointCloud2 observation
ObjectDetection[] objects
geometry_msgs/Transform camera_pose
---
ObjectLabel[] removed_labels
